identity-apps-core/apps/authentication-portal/src/main/webapp/oauth2_consent.jsp

```diff
@@ -61,8 +61,11 @@
                 // Get the displayName.
                 String displayName = (String) scope.get("displayName");
             
-                // Check if description exists, if not, use displayName.
-                String scopeName = scope.has("description") ? (String) scope.get("description") : displayName;
+                // Use optString to get description; it returns "" if the key is not found.
+                String description = scope.optString("description", "");
+
+                // Check if description is not empty, otherwise use displayName.
+                String scopeName = !description.isEmpty() ? description : displayName;
```

Comment:
Shall we use StringUtils for the empty check?  

Ref: https://stackoverflow.com/a/23419120/8810941